docs/source/checks/tabular/model_evaluation/plot_single_dataset_performance.py

```diff
@@ -3,8 +3,8 @@
 .. _plot_tabular_single_dataset_performance:
 
 Single Dataset Performance
-*****************************
-This notebooks provides an overview for using and understanding single dataset performance check.
+**********************
```

Comment:
Note that the line should not be shorter than the sentence above it

Reply:
did not know that. thanks